account.go

```diff
@@ -409,6 +409,10 @@ func (l *LegalEntity) AppendDetails(values *RequestValues) {
 		values.Add("legal_entity[business_vat_id]", l.BusinessVatID)
 	}
 
+	if l.Verification.Document != nil {
```

Comment:
Ack you're right. I was reading `Verification` off the `Account` struct instead of the `LegalEntity` struct (the former is a pointer but the latter isn't). That's fine then.